### DISCUSSION TITLE
Multiple Y axis on both sides of the screen

### DISCUSSION THREAD

**Discussion by froz3nice** [KoalaPlot/koalaplot-core#97] [Q&A] [answered]
Is there a way to create multiple y axis? Would be similar to this
![36LSv](https://github.com/user-attachments/assets/6d62eb89-bca5-4990-9877-3d9724f0250b)

**Comment by gsteckman** [comment, marked as answer]
It's not currently possible, and there is currently [this issue](https://github.com/KoalaPlot/koalaplot-core/issues/29) to track it. Please upvote it if you haven't already so we can keep track of what is most wanted.
